eng/pipelines/templates/steps/sync-directory.yml

```diff
@@ -81,12 +81,10 @@ steps:
     inputs:
       pwsh: true
       workingDirectory: ${{ parameters.WorkingDirectory }}
-      filePath: ${{ parameters.ScriptDirectory }}/Add-Issue-Comment.ps1
+      filePath: ${{ parameters.ScriptDirectory }}/Add-IssueComment.ps1
       arguments: >
         -RepoOwner "Azure"
         -RepoName "azure-sdk-tools"
         -IssueNumber "$(System.PullRequest.PullRequestNumber)"
-        -CommentPrefix "The following pipelines have been queued for testing:<br>"
-        -Comment "QueuedPipelines"
-        -CommentPostFix "<br>Sign off on the approval gate to test the release stage of each pipeline."
+        -Comment "The following pipelines have been queued for testing:<br>$env:QueuedPipelines<br>You can sign off on the approval gate to test the release stage of each pipeline."
```

Comment:
It is probably worth adding a link to the workflow in this common as well so that anyone new that submits a PR to the tools repo will have a link to the docs for the workflow.